src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/BannedAssembliesAnalyzer.cs

```diff
@@ -86,5 +89,15 @@ static void CheckType(ISymbolAnalysisContext context, ITypeSymbol type, ISymbol
                     break;
             }
         }
+
+        private static bool IsUtf8JsonReaderWriter(ITypeSymbol type)
+        {
+            return (type.Name == "Utf8JsonReader" || type.Name == "Utf8JsonWriter") && GetFullNamespace(type.ContainingNamespace) == "System.Text.Json";
```

Comment:
I am thinking there must be a way to get symbols for the reader and writer and then write this check as if (type == readerSymbol || type == writerSymbol).

Reply:
Isn't the way you mentioned "to get symbols for the reader and writer" what I'm doing now, be checking the name and namespace name to get the symbol?

Do you have a better way?

Reply:
The biggest issue right now is that how we get the symbol of those type of `Utf8JsonReader` as `INamedTypeSymbol`.

I know a way that if we have a compilation here, we could get the symbol by its full name, this could be a static readonly field, and then we could compare them by
```
SymbolEqualityComparer.Default.Equals(type, readerSymbol)
```

@KrzysztofCwalina do you know other alternatives of how we could fetch the symbol of a known type?